### PR TITLE
Update meta fields with creationTime and ttl

### PR DESCRIPTION
**Pull request by ggobugi27**

### CODE REVIEW

File: pact-lang-api.js [hunk 1]
```diff
@@ -313,23 +313,29 @@ var mkExp = function(pgmName) {
 };
 
 /**
- * Prepare a chainweb-style meta payload.
+ * Prepare a chainweb-style public meta payload.
  * @param sender {string} gas account
  * @param chainId {string} chain identifier
  * @param gasPrice {number} desired gas price
  * @param gasLimit {number} desired gas limit
+ * @param creationTime {number} desired creation time.
```
Comment (sirlensalot) [inline pact-lang-api.js]: should doco this as "time created (current time) as epoch milliseconds" <-- confirm with @emilypi that this is the intended value

**Review by sirlensalot — APPROVED** [review]
Small doco comment otherwise LGTM

File: pact-lang-api.js (hunk 1 above)
Comment (emilypi) [reply pact-lang-api.js]: Seconds since UNIX epoch, otherwise 👍

Comment (ggobugi27) [reply pact-lang-api.js]: changed to 
```
desired tx's time created (current time) as seconds
```

**Pull request merged**